maintainers/maintainer-list.nix

```diff
@@ -15893,4 +15893,10 @@
     github = "wuyoli";
     githubId = 104238274;
   };
+  ifd3f = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.